packages/types/src/index.ts

```diff
@@ -710,6 +683,78 @@ export type GradationMap = Record<
   { color: string; icon?: ReactElement; text?: string }
 >;
 
+export const ModeSettingTypes = {
+  CHECKBOX: "CHECKBOX",
+  DROPDOWN: "DROPDOWN",
+  SLIDER: "SLIDER"
+};
+
+export type DropdownOptions = {
+  default?: string;
+  label: string;
+  options: {
+    text: string;
+    value: string;
+    addTransportMode?: TransportMode;
```

Comment:
Sort attributes.

Reply:
On the topic of alphabetizing I have a fun VS Code plugin called "Alphabetize" which sorts lines really efficently